wasm/wasm.go

```diff
@@ -0,0 +1,15 @@
+package wasm
+
+import (
+	simappparams "github.com/cosmos/cosmos-sdk/simapp/params"
+	"github.com/strangelove-ventures/ibctest/v3/chain/cosmos"
+	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
+)
+
+func WasmEncoding() *simappparams.EncodingConfig {
```

Comment:
Since this is cosmos chain specific, might make sense for this wasm package to be chain/cosmos/wasm

Reply:
Yeah, that makes sense, I update that.